src/main/fc/fc_core.c

```diff
@@ -364,6 +370,15 @@ void tryArm(void)
                     pwmWriteDshotCommand(ALL_MOTORS, getMotorCount(), DSHOT_CMD_SPIN_DIRECTION_REVERSED, false);
                 }
             }
+        } else if (isModeActivationConditionPresent(BOXCONSTANTIDLE)) {
+            if (!(IS_RC_MODE_ACTIVE(BOXCONSTANTIDLE) || (tryingToArm == ARMING_DELAYED_CONSTANTIDLE))) {
+                constantIdleMode = false;
+            } else {
+                constantIdleMode = true;
+#ifdef USE_RUNAWAY_TAKEOFF
+                runawayTakeoffCheckDisabled = false;
+#endif
```

Comment:
There's no need to reset Runaway Takeoff Prevention for constant idle mode.  It's reset on crash flip because it's likely that the pilot crashed and props may be damaged which might lead to a runaway.  Constant idle is unlikely to cause a runaway risk.

Reply:
I was wondering as to why this was done. Makes good sense for crash flip, but not here.

Reply:
fixed